src/impl/vamp/bindings/common.hh

```diff
@@ -233,8 +260,8 @@ namespace vamp::binding
             return filter_robot_from_pointcloud<Robot>(pc, start, environment, point_radius);
         }
 
-        inline static auto eefk(const ConfigurationArray &start)
-            -> std::pair<std::array<float, 3>, std::array<float, 4>>
+        inline static auto
+        eefk(const ConfigurationArray &start) -> std::pair<std::array<float, 3>, std::array<float, 4>>
```

Comment:
Nit/not super relevant to this PR, but: we should rename this to `ee_fk`

Reply:
Should also really have a `Pose` type rather than the pair - but that one can be left to a new issue.